src/fontstash.h

```diff
@@ -811,6 +858,8 @@ void fonsClearState(FONScontext* stash)
 	state->font = 0;
 	state->blur = 0;
 	state->spacing = 0;
+	state->oversampleH = 2;
```

Comment:
Technically, I think it would be, but it could also be surprising. Let's be backwards compatible on this one.